### ISSUE TITLE
Can't change storage directory

### ISSUE THREAD

**Issue by JeicamPL** [BluewhaleRobot/systemback#18]
Hi,

I have problem with storage directory. I can't change it. All catalogs are red.

![1](https://user-images.githubusercontent.com/17877256/142850941-a457c605-8136-4d40-a38c-505bfee0031b.png)

I found on the internet that the filesystem must be writable by Linux, but my system is Ubuntu 20.04.3 and I have an ext4 filesystem, yet I can't change directory.

![2](https://user-images.githubusercontent.com/17877256/142851162-4d3123e0-b70c-464d-9c7e-2b396791b09f.png)

Overwriting the path in `/etc/systemback.conf` also doesn't help.
Does anyone know what the problem is?

Thanks for your help, best regards.

**Comment by Desmo5574** [comment]
I have the same problem as well. I've given up on all the forks of Systemback. The main dev stopped development for a reason and in his own words said the codebase "was a mess".  All of these forks are just workarounds/hacks to allow the software to run on newer versions of Ubuntu/Debian. At some point, they break and the current authors have no desire to deal with it. Looks like that is what is going on here. 
When Systemback was working flawlessly, 6-8 yrs ago, it was a godsend and what I used to create my snapshots. Well, it no longer works and I've since migrated to Timeshift. I highly recommend the rest of you do the same.
